examples/2_read_serial.cpp

```diff
@@ -48,8 +48,19 @@ int main()
         cout << "\n\t" << m.first;
     cout << '\n';
     cout << "Iteration 100 contains " << i.particles.size() << " particle species:";
-    for( auto const& ps : i.particles )
+    for( auto const& ps : i.particles ) {
         cout << "\n\t" << ps.first;
+        for( auto const& r : ps.second ) {
+            cout << "\n\t" << r.first;
+            cout << '\n';
+        }
+    }
+
+    std::pair<std::string, openPMD::ParticleSpecies> s_e = *i.particles.begin();
+    std::pair<std::string, openPMD::Record> s_c = *s_e.second.begin();
+    std::shared_ptr< double > charge = s_c.second[openPMD::RecordComponent::SCALAR].loadChunk<double>();
+    cout << "And first particle in particle species " << s_e.first
+         << " has " << s_c.first << " = " << charge.get()[0];
```

Comment:
The containers we access here can actually be unsorted and therefore it is not clear that the first species is always an electron and its first record is charge.
But we know the input file, therefore:
```suggestion
    openPMD::ParticleSpecies electrons = i.particles["electrons"];
    std::shared_ptr<double> charge = electrons["charge"][openPMD::RecordComponent::SCALAR].loadChunk<double>();
    cout << "And the first electron particle has a charge = " << charge.get()[0];
```

Reply:
Oh good point! It is true that it is unordered. Do you think it would be more important to ask for "electrons" or a first species, regardless of which it is?

Reply:
I think you are right, it is nicer to not generalize this completely!
Thanks @ax3l !!